dev-python/scikit-build/scikit-build-0.10.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Improved build system generator for Python C/C++/Fortran/Cython extensions"
+HOMEPAGE="https://github.com/scikit-build/scikit-build"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/distro[${PYTHON_USEDEP}]
+	dev-python/packaging[${PYTHON_USEDEP}]
+	dev-python/wheel[${PYTHON_USEDEP}]"
+
+DEPEND="test? (
+	dev-python/coverage[${PYTHON_USEDEP}]
+	dev-python/cython[${PYTHON_USEDEP}]
+	dev-python/flake8[${PYTHON_USEDEP}]
+	dev-python/path-py[${PYTHON_USEDEP}]
+	dev-python/pytest-mock[${PYTHON_USEDEP}]
+	dev-python/pytest-runner[${PYTHON_USEDEP}]
+	dev-python/pytest-virtualenv[${PYTHON_USEDEP}]
+	dev-python/PyQt5[testlib,${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	dev-python/virtualenv[${PYTHON_USEDEP}] )"
+
+PATCHES="${FILESDIR}/${P}-skip-sandbox-violation-tests.patch"
+
+distutils_enable_sphinx docs dev-python/sphinx_rtd_theme dev-python/sphinx-issues
```

Comment:
Something is missing, 
```
warning: cmd_build_py: byte-compiling is disabled, skipping.

UPDATING /tmp/portage/dev-python/scikit-build-0.10.0/work/scikit-build-0.10.0-python3_7/lib/skbuild/_version.py
set /tmp/portage/dev-python/scikit-build-0.10.0/work/scikit-build-0.10.0-python3_7/lib/skbuild/_version.py to '0.10.0'
 * python3_7: running distutils-r1_run_phase python_compile_all
Running Sphinx v1.7.5

Extension error:
Could not import extension cmake (exception: No module named 'cmake')
 * ERROR: dev-python/scikit-build-0.10.0::testworld failed (compile phase):
 *   (no error message)

 * The specific snippet of code:
 *       sphinx-build -b html -d "${dir}"/_build/doctrees "${dir}" "${dir}"/_build/html || die;
```

Reply:
Hmm, interesting, I thought this was working before, but apparently not.
I find no missing dependencies in 'requirements-docs' and google doesn't find a sphinx-cmake module. Therefore I have patched the conf.py file to fix this, it seems to work just fine like this.